public/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="app.css">
+    <title>Turn Based Game</title>
+</head>
+<body>
+
+<div class="game">
+	<div class="start"></div>
```

Comment:
These `start`, `battle`, and `end` divs will be destroyed if you do `$('.game').html(someVar)`.